website/docs/layouts/grid/partials/code/how-to-use.md

```diff
@@ -0,0 +1,373 @@
+
+```handlebars
+<Hds::Layout::Grid @columnMinWidth="33.33%" @gap="16">
+  <Doc::Placeholder @height="40px" @text="Item 1" @background="#e4e4e4" />
+  <Doc::Placeholder @height="40px" @text="Item 2" @background="#e4e4e4" />
+  <Doc::Placeholder @height="40px" @text="Item 3" @background="#e4e4e4" />
+  <Doc::Placeholder @height="40px" @text="Item 4" @background="#e4e4e4" />
+</Hds::Layout::Grid>
+```
+
+#### Using fixed values
+
+Column min-widths specified using pixels or other fixed units, allows you to create layouts which are “automatically” responsive.
+
+##### Grid within a wider view
+
+Narrow your browser window to see the responsive behavior.
+
+```handlebars
+<Hds::Layout::Grid @columnMinWidth="160px" @gap="16">
+  <Doc::Placeholder @height="40px" @text="Item 1" @background="#e4e4e4" />
+  <Doc::Placeholder @height="40px" @text="Item 2" @background="#e4e4e4" />
+  <Doc::Placeholder @height="40px" @text="Item 3" @background="#e4e4e4" />
+  <Doc::Placeholder @height="40px" @text="Item 4" @background="#e4e4e4" />
+</Hds::Layout::Grid>
+```
+
+##### The same grid within a narrower view
+
+At the specified column min width, columns are forced to stack in this narrower view.
+
+```handlebars
+<div class="doc-grid-mobile-view">
+  <Hds::Layout::Grid @columnMinWidth="160px" @gap="16">
+    <Doc::Placeholder @height="40px" @text="Item 1" @background="#e4e4e4" />
+    <Doc::Placeholder @height="40px" @text="Item 2" @background="#e4e4e4" />
+    <Doc::Placeholder @height="40px" @text="Item 3" @background="#e4e4e4" />
+    <Doc::Placeholder @height="40px" @text="Item 4" @background="#e4e4e4" />
+  </Hds::Layout::Grid>
+</div>
+```
+
+### Align
+
+Use the `@align` argument to align grid items to the "start", "end", "center" or "stretch" them within the grid parent.
+
+Note: The `Grid` parent will need a height set for the affect to be visible.
+
+```handlebars
+<div class="doc-grid-mobile-view">
+  <Hds::Layout::Grid @columnMinWidth="50px" @gap="16" @align="center" {{style height="100%"}}>
+    <Doc::Placeholder @height="40px" @text="Item 1" @background="#e4e4e4" />
+    <Doc::Placeholder @height="40px" @text="Item 2" @background="#e4e4e4" />
+    <Doc::Placeholder @height="40px" @text="Item 3" @background="#e4e4e4" />
+    <Doc::Placeholder @height="40px" @text="Item 4" @background="#e4e4e4" />
+  </Hds::Layout::Grid>
+</div>
+```
+
+### isInline
+
+To change the default display from `grid` to `inline-grid`, set `@isInline` to `true`.
+
+```handlebars
+Text before the inline grid.
+<Hds::Layout::Grid  @isInline={{true}} @tag="span" @gap="8">
+  <Doc::Placeholder @height="1.2em" @text="Item 1" @background="#e4e4e4" />
+  <Doc::Placeholder @height="1.2em" @text="Item 2" @background="#e4e4e4" />
+  <Doc::Placeholder @height="1.2em" @text="Item 3" @background="#e4e4e4" />
+  <Doc::Placeholder @height="1.2sem" @text="Item 4" @background="#e4e4e4" />
```

Comment:
```suggestion
  <Doc::Placeholder @height="1.2em" @text="Item 4" @background="#e4e4e4" />
```

Reply:
Oops. This example was updated previously to remove ems.